### ISSUE TITLE
Real time widget should maybe not include content tracking in the number of actions

### ISSUE THREAD

**Issue by tsteur** [piwik/piwik#12228]
I am referring to this number:

![image](https://user-images.githubusercontent.com/273120/32085353-eb2be19c-bb2a-11e7-8c11-c49a591c9566.png)

It looks like the number of content tracking requests is included in this number and would not really expect it. Especially since we

1) don't show them in the widget (afaik)
2) do not include them in other places like the overview: 
![image](https://user-images.githubusercontent.com/273120/32085378-1bde248a-bb2b-11e7-99bd-d0971a768046.png)

It is to be discussed how to handle events here but be great to maybe keep behaviour consistent and also not include them here. Sometimes a user would maybe want them included though (refs #9199). On the contrary, "max actions" and number of actions in a visit does seem to include events actually and I think the description for `3.9 actions (page views, downloads, outlinks and internal site searches) per visit` may be wrong and should mention events see https://github.com/piwik/piwik/blob/3.2.0/plugins/Actions/Columns/VisitTotalActions.php#L62-L68 (although not sure if this is used for it). So likely events could be actually counted.

When you do include content requests like currently, the number of action becomes a bit meaningless since there might be so many more content tracking requests than anything else. Eg here: 
![image](https://user-images.githubusercontent.com/273120/32085560-5fa68c6a-bb2c-11e7-8220-5500226ec7a5.png)
like 80% of actions are content tracking requests and it makes it look like there are like 31 actions per visit which is actually much less.

**Comment by mattab** [comment]
Absolutely, the number should be consistent with the Visitors > Overview > Actions. ie. only include (page views, downloads, outlinks, and internal site searches)

>  On the contrary, "max actions" and number of actions in a visit does seem to include events actually and I think the description for 3.9 actions (page views, downloads, outlinks and internal site searches) per visit may be wrong and should mention events see https://github.com/piwik/piwik/blob/3.2.0/plugins/Actions/Columns/VisitTotalActions.php#L62-L68 (although not sure if this is used for it). So likely events could be actually counted.

* Will be interesting to confirm whether events are included in this metric
* In general probably it's best to keep events out of this for now (until we have a distinction between interactive and non interactive events)

**Comment by sgiehl** [comment]
refs #11856
